test/mocks/overrides/override_auth_providers.dart

```diff
@@ -5,7 +5,7 @@ import "package:google_sign_in_mocks/google_sign_in_mocks.dart";
 import "package:hooks_riverpod/hooks_riverpod.dart";
 import "package:proxima/services/login_service.dart";
 
-import "../test_data/firebase_auth_user_mock.dart";
+import "../data/mock_firebase_auth_user.dart";
 
 final firebaseAuthMocksOverrides = [
```

Comment:
I don't think this is necessary for all providers, but I'm feeling this authentication providers need some small documentation to be very clear:
```suggestion
/// Override to use to test the login process
final firebaseAuthMocksOverrides = [
```